src/main/jib/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
this file and folder must be deleted.

in old version of the project file src/main/docker/entrypoint.sh is used

Reply:
done